.gitignore

```diff
@@ -1,5 +1,6 @@
 # IDE
 .idea/
+.vscode/
```

Comment:
#### Note

When using VSCode / VSCodium to develop on a Mac, you need to set the following on `vscode/settings.json` to get the Go plugin working correctly:

```yaml
{
    "go.toolsEnvVars": {
        "GOOS" : "windows"
    },
}
```